### PR TITLE
League stats support method test

### PR DESCRIPTION
**Pull request by ghost**
@zacisaacson @freeheeling @mcat56 Hey guys all the league stats support test are passing. I just need everyones approval to merge it into master. Thanks!!!

### CODE REVIEW

File: test/league_stats_support_test.rb [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+class LeagueStatsSupportTest < MiniTest::Test
+
+  def setup
+    locations = { games: './data/dummy_games.csv', teams: './data/teams.csv', game_teams: './data/dummy_game_teams.csv' }
+    @stat_tracker = StatTracker.from_csv(locations)
+  end
+
+  def test_generate_num_goals_per_team
+
+  example = {
```
Comment (freeheeling) [inline test/league_stats_support_test.rb]: Indent

File: test/league_stats_support_test.rb [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  def test_generate_allowed_goals
+    example = {
+      "6" => 7,
+      "3" => 11,
+      "9" => 5,
+      "8" => 9,
+      "24" => 9,
+      "20" => 15
+    }
+
```
Comment (freeheeling) [inline test/league_stats_support_test.rb:98]: Remove line

File: test/league_stats_support_test.rb [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  end
+
+  def test_generate_num_goals_per_team
+
+  example = {
+    "3" => 7,
+    "6" => 11,
+    "8" => 5,
+    "9" => 9,
+      "20" => 9,
```
Comment (mcat56) [inline test/league_stats_support_test.rb:20]: maybe make sure all indentation is consistent? Not sure if it just shows this way on Github

File: test/league_stats_support_test.rb [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+    assert_equal example, @stat_tracker.generate_home_and_away_goals
+  end
+
+  def test_highest_scoring_visitor
+    assert_equal "Real Salt Lake", @stat_tracker.highest_scoring_visitor
+  end
+
+  def test_highest_scoring_home_team
+  assert_equal "New York City FC", @stat_tracker.highest_scoring_home_team
```
Comment (freeheeling) [inline test/league_stats_support_test.rb]: Indent

File: test/league_stats_support_test.rb [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+  def test_lowest_scoring_visitor
+    assert_equal "New York Red Bulls", @stat_tracker.lowest_scoring_visitor
+  end
+
+  def test_lowest_scoring_home_team
+    assert_equal "Houston Dynamo", @stat_tracker.lowest_scoring_home_team
+  end
+
+  def test_generate_wins
+
```
Comment (freeheeling) [inline test/league_stats_support_test.rb]: Remove line

File: test/league_stats_support_test.rb [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+    away_wins = {
+      "3" => 0,
+      "6" => 2,
+      "8" => 0,
+      "9" => 0,
+      "20" => 0,
+      "24" => 3
+    }
+
```
Comment (freeheeling) [inline test/league_stats_support_test.rb:181]: Remove line

File: test/league_stats_support_test.rb [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+      "6" => 6,
+      "9" => 7,
+      "24" => 6,
+      "3" => 3,
+      "20" => 5,
+      "8" => 2
+    }
+
+    example = [home_goals, away_goals]
+
```
Comment (freeheeling) [inline test/league_stats_support_test.rb]: Remove line

**Review by mcat56 — APPROVED** [review]
Looks great!!!!

File: test/league_stats_support_test.rb [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+    "9" => 9,
+      "20" => 9,
+      "24" => 15
+    }
+
+    assert_equal example, @stat_tracker.generate_num_goals_per_team
+  end
+
+  def test_generate_num_games_per_team
+
```
Comment (freeheeling) [inline test/league_stats_support_test.rb]: Remove line

File: test/league_stats_support_test.rb [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+class LeagueStatsSupportTest < MiniTest::Test
+
+  def setup
+    locations = { games: './data/dummy_games.csv', teams: './data/teams.csv', game_teams: './data/dummy_game_teams.csv' }
+    @stat_tracker = StatTracker.from_csv(locations)
+  end
+
+  def test_generate_num_goals_per_team
+
```
Comment (freeheeling) [inline test/league_stats_support_test.rb]: Remove line

File: test/league_stats_support_test.rb [hunk 10]
```diff
@@ -0,0 +1,224 @@
+require_relative 'test_helper'
+require_relative '../lib/stat_tracker'
+require_relative '../lib/games'
+require_relative '../lib/teams'
+require_relative '../lib/game_teams'
+
+
```
Comment (freeheeling) [inline test/league_stats_support_test.rb]: Remove line

**Review by freeheeling — CHANGES_REQUESTED** [review]
Approved pending minor edits. Wunderbar, Sebastian!

File: test/league_stats_support_test.rb (hunk 10 above)
Comment (ghost) [reply test/league_stats_support_test.rb]: corrected, thank you sir

File: test/league_stats_support_test.rb (hunk 3 above)
Comment (ghost) [reply test/league_stats_support_test.rb:20]: Corrected, thank you ma'am

File: test/league_stats_support_test.rb (hunk 2 above)
Comment (ghost) [reply test/league_stats_support_test.rb:98]: corrected, thank you sir

File: test/league_stats_support_test.rb (hunk 6 above)
Comment (ghost) [reply test/league_stats_support_test.rb:181]: Done thank you

**Review by freeheeling — APPROVED** [review]
Thanks for making those changes.

**Pull request merged**